week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -72,7 +78,13 @@ function getEligibleStudents() {
   Hint: To complete the function, search how to change text to lower or upper case by using string method.
 */
 
-function getEligibleStudents2() {
+function getEligibleStudents2(arr) {
+  const newEligiblePupils = arr.filter(
+    (name) => name[0].charAt(0) === "A" || name[0].charAt(0) === "a"
```

Comment:
While using || is not wrong, but can ensure all sides are in specific case (capital or small) and do one comparison?

Reply:
Used toUpperCase() method to make sure that all cases begin with capital letters, and filtered "A" only.